[New Sample] Render Multilayer Symbols (#693)
Esri/arcgis-runtime-samples-java: symbology/render-multilayer-symbols/README.md

# Render multilayer symbols

> Show different kinds of multilayer symbols on a map similar to some pre-defined 2D simple symbol styles.

Reviewer: I find this sentence from the design overly complex: let's keep it short and to the point. The pre-existing symbols out the box from the API are mentioned later anyway.

```suggestion
Show different kinds of multilayer symbols on a map.
```

![Image of render multilayer symbols](RenderMultilayerSymbols.png)

## Use case

> Allows you to customize a graphic with a multilayer symbol. For example, you may want more customizable symbols than the one that is provided with the API to display a unique representation of a landmark.

Reviewer: ```suggestion
Allows you to customize a graphic with a multilayer symbol. For example, you may want more customizable symbols than those that are provided with the ArcGIS Runtime API in order to display a unique representation of a landmark.
```

## How to use the sample

> The sample loads multilayer symbols for points, polylines, and polygons.

Reviewer: ```suggestion
The sample loads with multilayer symbols displayed for points, polylines, and polygons.
```

## How it works

1. Create multilayer symbols for each predefined 2D simple symbol style.
    * For multilayer point symbols, use the `MultilayerPointSymbol` constructor.
    * For multilayer polyline symbols, use the `MultilayerPolylineSymbol` constructor.
    * For multilayer polygon symbols, use the `MultilayerPolygonSymbol` constructor.

> 2. Create graphics by passing in a geometry and the associated symbol.

Reviewer: ```suggestion
2. Create `Graphic`s by passing in a `Geometry` and the associated multilayer symbol.
```
